browser/ui/brave_ui_features.cc

```diff
@@ -5,9 +5,20 @@
 
 #include "brave/browser/ui/brave_ui_features.h"
 
+#include "base/feature_list.h"
+#include "build/build_config.h"
+
 namespace features {
 
 BASE_FEATURE(kBraveNtpSearchWidget,
              "BraveNtpSearchWidget",
              base::FEATURE_ENABLED_BY_DEFAULT);
-}
+
+#if BUILDFLAG(IS_WIN)
+// Enables window cloaking on window creation to prevent a white flash.
+BASE_FEATURE(kBraveWorkaroundNewWindowFlash,
+             "BraveWorkaroundNewWindowFlash",
+             base::FEATURE_DISABLED_BY_DEFAULT);
```

Comment:
disabled by default as the change may potentially break the browser startup. Enabling the feature via Griffin is safer as Griffin has fallback mechanism to revert the seed if multiple crashes was detected at startup.